charts/seed-bootstrap/dashboards/webhooks.json

```diff
@@ -0,0 +1,811 @@
+{
```

Comment:
Similar to controllers, I think it would be valuable to have one overview dashboard for all webhooks and one dashboard for drilling down into individual webhooks, e.g. with multiple percentiles for latency etc.

Reply:
Addressed in https://github.com/gardener/gardener/pull/7180/commits/989506bca5eee2e94d44ae13f35658668871d556